bin/test_with_coverage.dart

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:convert' show utf8, LineSplitter;
```

Comment:
[nit] sort imports within sections.

You should be able to use the analysis server to do it automatically in your IDE.

https://dart.dev/guides/language/effective-dart/style#do-sort-sections-alphabetically